classes/local/factor/object_factor_base.php

```diff
@@ -256,20 +265,24 @@ public function revoke_user_factor($factorid) {
      * @return bool
      */
     public function has_lastverified() {
-        return false;
+        return true;
```

Comment:
I'm wondering whether this public api is now completely redundant? All factors will now always be able to be verified